terraform/itap-piem.alpha-canada.ca.tf

```diff
@@ -0,0 +1,9 @@
+resource "aws_route53_record" "itap-piem-alpha-canada.ca-A" {
```

Comment:
Left one in there.

Reply:
Man I even had 2 coffees this morning.  Hopefully this will do it